identity-wallet/src/state/reducers/authorization.rs

```diff
@@ -200,9 +205,14 @@ pub async fn handle_siopv2_authorization_request(state: AppState, _action: Actio
             connection.last_interacted = connection_time.clone();
         });
 
+    // TODO: This is a HORRIBLE solution to determine the connection_id by the non-unique "issuer name".
+    // It is a TEMPORARY solution and should only be used in DEMO environments,
+    // since we currently lack a unique identitfier to distinguish connections.
+    let connection_id = base64::encode_config(&client_name, base64::URL_SAFE);
+
     if result.is_none() {
         connections.push(Connection {
-            id: "TODO".to_string(),
+            id: connection_id.clone(),
```

Comment:
Is there any implication of now using a `connection_id` here, but not in `fn handle_oid4vp_authorization_request`:
```rust
        if result.is_none() {
            connections.push(Connection {
                id: "TODO".to_string(),
                client_name,
                url: connection_url,
                verified: false,
                first_interacted: connection_time.clone(),
                last_interacted: connection_time,
            })
        };
```